src/write_hdf5_xml/write_hdf5_xml.c

```diff
@@ -2321,7 +2343,10 @@ int main(int argc, char **argv)
 
     // Incidence angle map
     split_dir_and_file(params->incidence_angle_file, directory, filename);
-    meta = meta_read(params->rtc_HH_metadata);
+    if (params->rtc_HH_metadata)
+      meta = meta_read(params->rtc_HH_metadata);
+    else if (params->rtc_VV_metadata)
+      meta = meta_read(params->rtc_VV_metadata);
```

Comment:
Here, too: error out if no metadata can be read
else {
asfPrintError("No metadata file found for incidence angle map!");
meta = NULL;